charts/celeborn/templates/master-statefulset.yaml

```
@@ -71,6 +71,22 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+      {{- $dirs := .Values.volumes.master }}
+      {{- if eq "hostPath" (index $dirs 0).type }}
+      - name: chown-celeborn-master-volume
+        image: alpine:3.18.0
```

we may want the latest patched version if possible
```suggestion
        image: alpine:3.18
```